src/fiatconnect/slice.ts

```diff
@@ -271,6 +276,17 @@ export const slice = createSlice({
         status: SendingTransferStatus.Completed,
       }
     },
+    createFiatConnectTransferTxProcessing: (
+      state,
+      action: PayloadAction<CreateFiatConnectTransferTxProcessingAction>
+    ) => {
+      state.transfer = {
+        quoteId: action.payload.quoteId,
+        flow: action.payload.flow,
+        txHash: null,
```

Comment:
There may be a hash associated with the TX, but even if there is, I am not sure if it's actually accessible through the error object we get. I included it in the action payload because it's supported by the pending screen if we set one, but I can remove it for the time being.